docs/topics/auth/default.txt

```diff
@@ -1280,9 +1280,19 @@ implementation details see :ref:`using-the-views`.
     Allows a user to reset their password by generating a one-time use link
     that can be used to reset the password, and sending that link to the
     user's registered email address.
-
-    If the email address provided does not exist in the system, this view
-    won't send an email, but the user won't receive any error message either.
+    
+    This view will send an email only if the following conditions are met:
```

Comment:
```suggestion
    This view will send an email if the following conditions are met:
```
Mainly, I just think "only if" doesn't sound very natural.

But also, "only if" here doesn't technically tell us when an email definitely will be sent (which is the important thing we're trying to say at this point). You could say "if and only if", but I think since the opposite implication is spelt out explicitly in the sentence below, simply an "if" would be better here.

Reply:
Changed in 808d1b8593.